core-utilities/src/main/proto/TextAnnotation.proto

```diff
@@ -0,0 +1,143 @@
+syntax = "proto2";
```

Comment:
Do you want to add a few lines comments on top of this files saying, what is is, how it's generated, and when you will need to change it?